### PR TITLE
Add music page

### PR DESCRIPTION
**Pull request by tomduckering**
Includes releases and resources.

Added resources as static files. Seems to make sense.

Badges for spotify and apple music are broken right now - because they
are SVGs

tweaked the stylesheet to make this work more easily.

### CODE REVIEW

File: src/pages/music.tsx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+                        title
+                        files {
+                            name
+                            url
+                        }
+                    }
+                }
+            }
+        }
+    `)
```
Comment (mauriceyap) [inline src/pages/music.tsx:94]: could we extract this query out of the component function? Will allow for mocking for easier testing and previews in future if we decide to go that way (Apollo has nice mocking stuff).

File: src/pages/music.tsx [hunk 2]
```diff
@@ -0,0 +1,222 @@
+import React from "react"
+import { graphql, useStaticQuery } from "gatsby"
+import Img from "gatsby-image/withIEPolyfill"
+
+import Layout from "../components/layout"
+import Hero from "../components/hero"
+import Bio from "../components/bio"
+
+const Students: React.FC<{}> = () => {
```
Comment (mauriceyap) [inline src/pages/music.tsx]: `MusicPage`

Comment (tomduckering) [reply src/pages/music.tsx]: Good spot

File: src/pages/music.tsx (hunk 1 above)
Comment (tomduckering) [reply src/pages/music.tsx:94]: Agree that would be a good refactor. Will hold off on that for now though.

**Comment by tomduckering** [comment]
The build is failing for complaints on other files. Will sort hat in another PR.

**Pull request merged**